src/main/scala/com/yotpo/metorikku/utils/FileUtils.scala

```diff
@@ -71,4 +72,13 @@ object FileUtils {
     val reader = new BufferedReader(new InputStreamReader(fsFile))
     reader.lines.collect(Collectors.joining)
   }
+
+  def removeFileWithHadoop(path: String, sparkSession: SparkSession): AnyVal = {
+    val file = new Path(path)
+    val hadoopConf = sparkSession.sessionState.newHadoopConf()
+    val fs = FileSystem.get(hadoopConf)
+    if (fs.exists(file)){
+      fs.delete(file, true)
```

Comment:
Are you expecting to get the path for the .hudi folder? wouldn't the fs.delete(file, true) run a recursive search and bring the files from the aux/archive folders as well? if you're not looking for recursive search then you shouldn't pass the true